template.json

```diff
@@ -21,12 +21,20 @@
       "eslint-plugin-react-hooks": "4.2.0",
       "eslint-plugin-jest": "24.3.6",
       "@typescript-eslint/eslint-plugin": "4.22.1",
-      "@typescript-eslint/parser": "4.22.1"
+      "@typescript-eslint/parser": "4.22.1",
+      "node-sass": "4.14.1",
+      "stylelint": "13.13.1",
+      "stylelint-config-sass-guidelines": "8.0.0",
+      "stylelint-scss": "3.19.0",
+      "stylelint-order": "4.1.0",
+      "stylelint-config-property-sort-order-smacss": "7.1.0"
     },
     "scripts": {
       "lint": "eslint ./src --ext .ts,.tsx",
       "lint:fix": "eslint ./src --ext .ts,.tsx --fix",
-      "prettier": "prettier \"src/**/*.{ts,html}\" --write"
+      "prettier": "prettier \"src/**/*.{ts,html}\" --write",
+      "stylelint": "stylelint '**/*.scss'",
+      "stylelint:fix": "stylelint '**/*.scss' --fix"
```

Comment:
@rafayet-monon As we have multiple linting and also prettier here How about adding a script to run all of them at once? For example
```json
    "codebase": "yarn lint && yarn prettier && yarn stylelint",
    "codebase:fix": "yarn lint:fix && yarn prettier && yarn stylelint:fix",
```